src/pages/Version.tsx

```diff
@@ -31,10 +31,12 @@ import { PageDoesNotExist } from "pages/404";
 import { shortenGithash, githubPRLinkify } from "utils/string";
 import { jiraLinkify } from "utils/string/jiraLinkify";
 import VersionPageBreadcrumbs from "./version/Breadcrumbs";
-import { BuildVariants } from "./version/BuildVariants";
-import { ActionButtons } from "./version/index";
-import { Metadata } from "./version/Metadata";
-import { Tabs } from "./version/Tabs";
+import {
+  ActionButtons,
+  BuildVariants,
+  Metadata,
+  VersionTabs,
+} from "./version/index";
```

Comment:
an import from ./version should work the same way?
```suggestion
} from "./version";
```

Reply:
It doesn't work 😟... I think it might be getting confused between `Version` and `version`